src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -1807,8 +1807,7 @@ bool LibraryCallKit::inline_vector_test() {
                                                  opd1->bottom_type()->isa_vectmask())) {
     opd2 = opd1;
   }
-  Node* cmp = new VectorTestNode(opd1, opd2, booltest);
-  cmp = gvn().transform(cmp);
+  Node* cmp = gvn.transform(new VectorTestNode(opd1, opd2, booltest));
```

Comment:
should be `gvn()` instead of `gvn` ?

Reply:
Fixed, the remaining failure in GHA seems to be JDK-8292498